quick-start/2.-hosted-services.md

```diff
@@ -172,13 +172,21 @@ Which gets us a list of network peers:
 /dns4/kras-09.fluence.dev/tcp/19001/wss/p2p/12D3KooWD7CvsYcpF9HE9CCV9aY3SJ317tkXVykjtZnht2EbzDPm
 ```
 
-Let's use the peer`12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi` as our deployment target and deploy our service from the VSCode terminal. In the `quickstart/2-hosted-services` directory run:
+Let's use the peer`12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi` as our deployment target and deploy our service from the VSCode terminal. 
 
+To do so we first need to generate a `secret key`. Run:
 ```bash
-aqua remote deploy \
-     --addr /dns4/kras-04.fluence.dev/tcp/19001/wss/p2p/12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi \
-     --config-path configs/hello_world_deployment_cfg.json \
-     --service hello-world
+aqua key create
+```
+
+Replace `<YOUR_SECRET_KEY>` with the generated secret key and while in the `quickstart/2-hosted-services` directory run:
+
+```bash
+aqua remote deploy_service \
+    --addr /dns4/kras-04.fluence.dev/tcp/19001/wss/p2p/12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi \
+    --sk <YOUR_SECRET_KEY> \
+    --data-path configs/hello_world_deployment_cfg.json \
```

Comment:
I believe it should be `--config-path`.

Reply:
If I try to deploy a service using `--config-path` I get `Unexpected option: --config-path`

Reply:
It seems we have different versions of the `aqua`:
```
$ aqua --version
0.7.2-307
```

Compare with version list at https://www.npmjs.com/package/@fluencelabs/aqua

Reply:
it is `--config-path` and yes, `--sk` is not optional anymore. @Azephiar please update and we can merge.